openapi/src/parts/controlled_azure_storage_container.yaml

```diff
@@ -24,6 +24,32 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/workspaces/v1/{workspaceId}/resources/controlled/azure/storageContainer/{resourceId}/clone:
```

Comment:
I would expect to see a `.../clone/result` endpoint to complete the async endpoint implementation.

Reply:
There is also a general-purpose job result endpoint. Why do we need endpoints specific to each resource type?

`/api/job/v1/jobs/{jobId}`.

Reply:
The requirements for async endpoints are defined in this doc: [MC Terra - Async REST API Specification](https://docs.google.com/document/d/1PTd4xvmV9xnEkWaIgFc6d3VUyJwwymQTFdhpKrdsUKw/edit).

The specific result endpoint provides a type-safe interface. The `jobs` API is to let us write generic waiting code. It does not return the result object. It refers to the resultURL, which is the type-safe endpoint, for retrieving the actual result body.

Reply:
Will add.